lib/active_record_doctor/detectors/missing_non_null_constraint.rb

```diff
@@ -18,8 +18,14 @@ class MissingNonNullConstraint < Base # :nodoc:
 
       private
 
+      TIMESTAMPS = ["created_at", "created_on", "updated_at", "updated_on"].freeze
+
       def message(column:, table:)
-        "add `NOT NULL` to #{table}.#{column} - models validates its presence but it's not non-NULL in the database"
+        if self.class::TIMESTAMPS.include?(column)
```

Comment:
```suggestion
        if TIMESTAMPS.include?(column)
```

Reply:
@fatkodima Thanks again for the quick review! I made this change in the latest revision.